modules/ROOT/pages/introduction/intro-config-use-studio.adoc

```diff
@@ -0,0 +1,159 @@
++
+image::introduction/intro-config-studio-select-listener.png[Select HTTP Listener as an Input Source]
++
+. Drag the *Listener* operation onto the canvas.
+. In the *Listener* configuration screen, click the plus sign (*+*) next to the *Connector configuration* field to add a global element for HTTP Listener.
++
+The following screenshot shows the default values for the HTTP Listener global element:
++
+image::introduction/intro-config-studio-listener.png[Global Element Screen for HTTP Listener]
++
+. Accept the defaults on the *General* tab or specify required information:
+* To listen on a specific port for every network interface on the server, leave the default value `0.0.0.0` for the *Host* property.
+* To listen for incoming requests generated inside the server, set the *Port* value to `localhost`.
+. To use TLS, access the *TLS* tab and specify the required information.
+. On the *Advanced* tab, specify a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specified server.
+. Click *OK*.
+. Set the *Path* field to the appropriate path.
+. Select the other tabs and specify the appropriate field values.
+
+For more information about HTTP Listener, see the xref:http/http-documentation.adoc[HTTP Connector Reference].
+
+=== Use Scheduler as an Input Source
+
+Use Scheduler to trigger a flow when a time-based condition is met. You can configure Scheduler to trigger a flow at a regular interval or you can use a more flexible cron expression. For example, you can use Scheduler to trigger a flow every five seconds, or you can set up a cron expression to trigger a flow at noon everyday for one month.
+
+To use Scheduler as an input source:
+
+. In the Mule Palette view, search for and select *Scheduler*:
++
+image::introduction/intro-config-studio-select-scheduler.png[Select Scheduler as an Input Source]
++
+. Drag *Scheduler* onto the canvas.
++
+The following screenshot shows the default values for the scheduling strategy:
++
+image::introduction/intro-config-studio-scheduler.png[Default Values for the Scheduling Strategy]
++
+. In the *Scheduling Strategy* field of the *Scheduler configuration* screen, select whether you want to use a fixed frequency or cron expression to trigger the flow
+* If you select *Fixed Frequency*, enter values for these fields:
++
+** *Frequency*
++
+The frequency at which the Scheduler triggers the flow, based on the time unit specified in the *Time unit* field.
++
+** *Start delay*
++
+The amount of time to wait before triggering the flow for the first time after the app starts, based on the time unit specified in the *Time unit* field.
++
+** *Time unit*
++
+The time unit for the values of *Frequency* and *Start Delay*.
++
+* If you select *Cron*, enter the cron expression in the *Expression* field and the time zone in the *Time Zone* field. For
+ the format and valid values for the *Time Zone* field, see the Javadoc for the Java TimeZone class.
```

Comment:
link?

Reply:
A good suggestion, but I am seeing lots of places this is documented, and don't know which specific one to refer to.